src/Common/Doctrine/Type/LocaleType.php

```diff
@@ -31,7 +31,7 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
      */
     public function convertToPHPValue($locale, AbstractPlatform $platform)
     {
-        if (APPLICATION === 'frontend') {
+        if (APPLICATION === 'Frontend') {
```

Comment:
There isn't a constant for this at the moment and since the application constant is something that needs to be refactored I would leave it like this for now